src/utils/formatDate.jsx

```diff
@@ -0,0 +1,5 @@
+export function formatDate(data) {
```

Comment:
It's called unixTimestamp, so don't name it `data` when you can provide a better naming